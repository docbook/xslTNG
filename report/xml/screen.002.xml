<?xml version="1.0" encoding="utf-8"?>
<article xmlns="http://docbook.org/ns/docbook" version="5.0">
<title>Article wrapper</title>

<para>This is a screen.</para>

<screen>Code 1
Code 2
Code 3
Code 4
Code 5
</screen>

<para>So is this.</para>

<screen>Code 1
Code 2
Code 3
Code 4
Code 5
</screen>

<para>The next screen is a continuation of the previous.</para>

<screen continuation="continues">Code 6
Code 7
Code 8
Code 9
Code 10
</screen>

</article>
