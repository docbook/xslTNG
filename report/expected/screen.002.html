<!DOCTYPE html><html xmlns="http://www.w3.org/1999/xhtml" class="no-js"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8"/><script>(function(H){H.className=H.className.replace(/\bno-js\b/,'js')})(document.documentElement)</script><title>Article wrapper</title><meta name="viewport" content="width=device-width, initial-scale=1.0"/><link href="https://purl.org/dc/elements/1.1/" rel="schema.dc"/><meta content="2011-04-22T17:02:00-06:00" name="dc.modified"/><meta content="DocBook xslTNG" name="generator"/><link href="./css/docbook.css" rel="stylesheet" media="screen"/><script src="./js/copy-verbatim.js" defer="defer"></script></head><body class="home"><nav class="top"></nav><main><article class="article component"><header><h1>Article wrapper</h1></header><p>This is a screen.</p><div class="pre-wrap"><pre class="screen verbatim verblines"><span class="line" db-line="1"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 1</code></span></span>
<span class="line" db-line="2"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 2</code></span></span>
<span class="line" db-line="3"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 3</code></span></span>
<span class="line" db-line="4"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 4</code></span></span>
<span class="line" db-line="5"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 5</code></span></span>
</pre></div><p>So is this.</p><div class="pre-wrap"><pre class="screen verbatim verblines"><span class="line" db-line="1"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 1</code></span></span>
<span class="line" db-line="2"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 2</code></span></span>
<span class="line" db-line="3"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 3</code></span></span>
<span class="line" db-line="4"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 4</code></span></span>
<span class="line" db-line="5"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 5</code></span></span>
</pre></div><p>The next screen is a continuation of the previous.</p><div class="pre-wrap"><pre class="screen verbatim verblines"><span class="line" db-line="1"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 6</code></span></span>
<span class="line" db-line="2"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 7</code></span></span>
<span class="line" db-line="3"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 8</code></span></span>
<span class="line" db-line="4"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 9</code></span></span>
<span class="line" db-line="5"><span class="ln">  <span class="nsep">|</span></span><span class="ld"><code>Code 10</code></span></span>
</pre></div></article></main><nav class="bottom"></nav></body></html>